app/.gitignore

```diff
@@ -1,69 +1,69 @@
-# Logs
-logs
-*.log
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
+# Project specifics
+
+## dotenv environment variable files
+.env*
+
+## gatsby files
+.cache/
+public
+
+
+# IDEs
+*.swp
+
+# OS specifics
 
-# Runtime data
+## Runtime data
 pids
 *.pid
 *.seed
 *.pid.lock
 
-# Directory for instrumented libs generated by jscoverage/JSCover
-lib-cov
+## MacOS
```

Comment:
It's okay for take the code of MacOS?

Reply:
I guess so. Contents of the file is originally written by Gatsby's original template, and I only removed a few lines which I know for sure. There're bunch of files I don't recognize so I kept them there for the sake of safety. 😄